Doc/library/logging.rst

```diff
@@ -1111,6 +1111,12 @@ functions.
    .. note:: If you are thinking of defining your own levels, please see the
       section on :ref:`custom-levels`.
 
+.. function:: getLevelNamesDict()
+
+   Returns the dictionary of all available level names and respective logging levels.
```

Comment:
I think the description should say it's a mapping from level names to their corresponding logging levels. Just saying it's a dict doesn't give enough context. Also, I think a better name would be `getLevelNamesMapping()`, but I feel less strongly about that.

And if this returns a copy of the dict, then it should be noted here. My initial expectation was that it would always return the same dict.